### ISSUE TITLE
KHM - Kaldheim Set Card Implementation Tracking 

### ISSUE THREAD

**Issue by theelk801** [magefree/mage#7248]
Mark cards you're working on in the second post of this issue.
If you can't check things off, leave a comment instead.
Once the card is committed, check off the card here to show that it's finished.
Don't worry about adding new spoilers to this list, it's generated by a script.

[All Sets](https://github.com/magefree/mage/wiki/Set-implementation-list)

# Unimplemented Cards

# Implemented Cards
<details><summary>Click to expand</summary>

- [x] Done -- [Absorb Identity](https://scryfall.com/search?q=!"Absorb+Identity")
- [x] Done -- [Aegar, the Freezing Flame](https://scryfall.com/search?q=!"Aegar,+the+Freezing+Flame")
- [x] Done -- [Alpine Meadow](https://scryfall.com/search?q=!"Alpine+Meadow")
- [x] Done -- [Alrund's Epiphany](https://scryfall.com/search?q=!"Alrunds+Epiphany")
- [x] Done -- [Alrund, God of the Cosmos](https://scryfall.com/search?q=!"Alrund,+God+of+the+Cosmos")
- [x] Done -- [Annul](https://scryfall.com/search?q=!"Annul")
- [x] Done -- [Arachnoform](https://scryfall.com/search?q=!"Arachnoform")
- [x] Done -- [Arctic Treeline](https://scryfall.com/search?q=!"Arctic+Treeline")
- [x] Done -- [Armed and Armored](https://scryfall.com/search?q=!"Armed+and+Armored")
- [x] Done -- [Arni Brokenbrow](https://scryfall.com/search?q=!"Arni+Brokenbrow")
- [x] Done -- [Arni Slays the Troll](https://scryfall.com/search?q=!"Arni+Slays+the+Troll")
- [x] Done -- [Ascendant Spirit](https://scryfall.com/search?q=!"Ascendant+Spirit")
- [x] Done -- [Ascent of the Worthy](https://scryfall.com/search?q=!"Ascent+of+the+Worthy")
- [x] Done -- [Augury Raven](https://scryfall.com/search?q=!"Augury+Raven")
- [x] Done -- [Avalanche Caller](https://scryfall.com/search?q=!"Avalanche+Caller")
- [x] Done -- [Axgard Armory](https://scryfall.com/search?q=!"Axgard+Armory")
- [x] Done -- [Axgard Braggart](https://scryfall.com/search?q=!"Axgard+Braggart")
- [x] Done -- [Axgard Cavalry](https://scryfall.com/search?q=!"Axgard+Cavalry")
- [x] Done -- [Barkchannel Pathway](https://scryfall.com/search?q=!"Barkchannel+Pathway")
- [x] Done -- [Basalt Ravager](https://scryfall.com/search?q=!"Basalt+Ravager")
- [x] Done -- [Battershield Warrior](https://scryfall.com/search?q=!"Battershield+Warrior")
- [x] Done -- [Battle Mammoth](https://scryfall.com/search?q=!"Battle+Mammoth")
- [x] Done -- [Battle for Bretagard](https://scryfall.com/search?q=!"Battle+for+Bretagard")
- [x] Done -- [Battle of Frost and Fire](https://scryfall.com/search?q=!"Battle+of+Frost+and+Fire")
- [x] Done -- [Battlefield Raptor](https://scryfall.com/search?q=!"Battlefield+Raptor")
- [x] Done -- [Bearded Axe](https://scryfall.com/search?q=!"Bearded+Axe")
- [x] Done -- [Behold the Multiverse](https://scryfall.com/search?q=!"Behold+the+Multiverse")
- [x] Done -- [Berg Strider](https://scryfall.com/search?q=!"Berg+Strider")
- [x] Done -- [Beskir Shieldmate](https://scryfall.com/search?q=!"Beskir+Shieldmate")
- [x] Done -- [Bind the Monster](https://scryfall.com/search?q=!"Bind+the+Monster")
- [x] Done -- [Binding the Old Gods](https://scryfall.com/search?q=!"Binding+the+Old+Gods")
- [x] Done -- [Birgi, God of Storytelling](https://scryfall.com/search?q=!"Birgi,+God+of+Storytelling")
- [x] Done -- [Blessing of Frost](https://scryfall.com/search?q=!"Blessing+of+Frost")
- [x] Done -- [Blightstep Pathway](https://scryfall.com/search?q=!"Blightstep+Pathway")
- [x] Done -- [Blizzard Brawl](https://scryfall.com/search?q=!"Blizzard+Brawl")
- [x] Done -- [Blood on the Snow](https://scryfall.com/search?q=!"Blood+on+the+Snow")
- [x] Done -- [Bloodline Pretender](https://scryfall.com/search?q=!"Bloodline+Pretender")
- [x] Done -- [Bloodsky Berserker](https://scryfall.com/search?q=!"Bloodsky+Berserker")
- [x] Done -- [Boreal Outrider](https://scryfall.com/search?q=!"Boreal+Outrider")
- [x] Done -- [Bound in Gold](https://scryfall.com/search?q=!"Bound+in+Gold")
- [x] Done -- [Breakneck Berserker](https://scryfall.com/search?q=!"Breakneck+Berserker")
- [x] Done -- [Bretagard Stronghold](https://scryfall.com/search?q=!"Bretagard+Stronghold")
- [x] Done -- [Brinebarrow Intruder](https://scryfall.com/search?q=!"Brinebarrow+Intruder")
- [x] Done -- [Broken Wings](https://scryfall.com/search?q=!"Broken+Wings")
- [x] Done -- [Burning-Rune Demon](https://scryfall.com/search?q=!"Burning-Rune+Demon")
- [x] Done -- [Calamity Bearer](https://scryfall.com/search?q=!"Calamity+Bearer")
- [x] Done -- [Canopy Tactician](https://scryfall.com/search?q=!"Canopy+Tactician")
- [x] Done -- [Cinderheart Giant](https://scryfall.com/search?q=!"Cinderheart+Giant")
- [x] Done -- [Clarion Spirit](https://scryfall.com/search?q=!"Clarion+Spirit")
- [x] Done -- [Cleaving Reaper](https://scryfall.com/search?q=!"Cleaving+Reaper")
- [x] Done -- [Codespell Cleric](https://scryfall.com/search?q=!"Codespell+Cleric")
- [x] Done -- [Colossal Plow](https://scryfall.com/search?q=!"Colossal+Plow")
- [x] Done -- [Cosima, God of the Voyage](https://scryfall.com/search?q=!"Cosima,+God+of+the+Voyage")
- [x] Done -- [Cosmos Charger](https://scryfall.com/search?q=!"Cosmos+Charger")
- [x] Done -- [Cosmos Elixir](https://scryfall.com/search?q=!"Cosmos+Elixir")
- [x] Done -- [Craven Hulk](https://scryfall.com/search?q=!"Craven+Hulk")
- [x] Done -- [Crippling Fear](https://scryfall.com/search?q=!"Crippling+Fear")
- [x] Done -- [Crush the Weak](https://scryfall.com/search?q=!"Crush+the+Weak")
- [x] Done -- [Cyclone Summoner](https://scryfall.com/search?q=!"Cyclone+Summoner")
- [x] Done -- [Darkbore Pathway](https://scryfall.com/search?q=!"Darkbore+Pathway")
- [x] Done -- [Deathknell Berserker](https://scryfall.com/search?q=!"Deathknell+Berserker")
- [x] Done -- [Demon Bolt](https://scryfall.com/search?q=!"Demon+Bolt")
- [x] Done -- [Demonic Gifts](https://scryfall.com/search?q=!"Demonic+Gifts")
- [x] Done -- [Depart the Realm](https://scryfall.com/search?q=!"Depart+the+Realm")
- [x] Done -- [Disdainful Stroke](https://scryfall.com/search?q=!"Disdainful+Stroke")
- [x] Done -- [Divine Gambit](https://scryfall.com/search?q=!"Divine+Gambit")
- [x] Done -- [Dogged Pursuit](https://scryfall.com/search?q=!"Dogged+Pursuit")
- [x] Done -- [Doomskar Oracle](https://scryfall.com/search?q=!"Doomskar+Oracle")
- [x] Done -- [Doomskar Titan](https://scryfall.com/search?q=!"Doomskar+Titan")
- [x] Done -- [Doomskar](https://scryfall.com/search?q=!"Doomskar")
- [x] Done -- [Dragonkin Berserker](https://scryfall.com/search?q=!"Dragonkin+Berserker")
- [x] Done -- [Draugr Necromancer](https://scryfall.com/search?q=!"Draugr+Necromancer")
- [x] Done -- [Draugr Recruiter](https://scryfall.com/search?q=!"Draugr+Recruiter")
- [x] Done -- [Draugr Thought-Thief](https://scryfall.com/search?q=!"Draugr+Thought-Thief")
- [x] Done -- [Draugr's Helm](https://scryfall.com/search?q=!"Draugrs+Helm")
- [x] Done -- [Dread Rider](https://scryfall.com/search?q=!"Dread+Rider")
- [x] Done -- [Dream Devourer](https://scryfall.com/search?q=!"Dream+Devourer")
- [x] Done -- [Dual Strike](https://scryfall.com/search?q=!"Dual+Strike")
- [x] Done -- [Duskwielder](https://scryfall.com/search?q=!"Duskwielder")
- [x] Done -- [Dwarven Hammer](https://scryfall.com/search?q=!"Dwarven+Hammer")
- [x] Done -- [Dwarven Reinforcements](https://scryfall.com/search?q=!"Dwarven+Reinforcements")
- [x] Done -- [Egon, God of Death](https://scryfall.com/search?q=!"Egon,+God+of+Death")
- [x] Done -- [Elderfang Disciple](https://scryfall.com/search?q=!"Elderfang+Disciple")
- [x] Done -- [Elderfang Ritualist](https://scryfall.com/search?q=!"Elderfang+Ritualist")
- [x] Done -- [Elderleaf Mentor](https://scryfall.com/search?q=!"Elderleaf+Mentor")
- [x] Done -- [Elven Ambush](https://scryfall.com/search?q=!"Elven+Ambush")
- [x] Done -- [Elven Bow](https://scryfall.com/search?q=!"Elven+Bow")
- [x] Done -- [Elvish Warmaster](https://scryfall.com/search?q=!"Elvish+Warmaster")
- [x] Done -- [Eradicator Valkyrie](https://scryfall.com/search?q=!"Eradicator+Valkyrie")
- [x] Done -- [Esika's Chariot](https://scryfall.com/search?q=!"Esikas+Chariot")
- [x] Done -- [Esika, God of the Tree](https://scryfall.com/search?q=!"Esika,+God+of+the+Tree")
- [x] Done -- [Faceless Haven](https://scryfall.com/search?q=!"Faceless+Haven")
- [x] Done -- [Fall of the Impostor](https://scryfall.com/search?q=!"Fall+of+the+Impostor")
- [x] Done -- [Fearless Liberator](https://scryfall.com/search?q=!"Fearless+Liberator")
- [x] Done -- [Fearless Pup](https://scryfall.com/search?q=!"Fearless+Pup")
- [x] Done -- [Feed the Serpent](https://scryfall.com/search?q=!"Feed+the+Serpent")
- [x] Done -- [Fire Giant's Fury](https://scryfall.com/search?q=!"Fire+Giants+Fury")
- [x] Done -- [Firja's Retribution](https://scryfall.com/search?q=!"Firjas+Retribution")
- [x] Done -- [Firja, Judge of Valor](https://scryfall.com/search?q=!"Firja,+Judge+of+Valor")
- [x] Done -- [Forging the Tyrite Sword](https://scryfall.com/search?q=!"Forging+the+Tyrite+Sword")
- [x] Done -- [Frenzied Raider](https://scryfall.com/search?q=!"Frenzied+Raider")
- [x] Done -- [Frost Augur](https://scryfall.com/search?q=!"Frost+Augur")
- [x] Done -- [Frost Bite](https://scryfall.com/search?q=!"Frost+Bite")
- [x] Done -- [Frostpeak Yeti](https://scryfall.com/search?q=!"Frostpeak+Yeti")
- [x] Done -- [Frostpyre Arcanist](https://scryfall.com/search?q=!"Frostpyre+Arcanist")
- [x] Done -- [Funeral Longboat](https://scryfall.com/search?q=!"Funeral+Longboat")
- [x] Done -- [Fynn, the Fangbearer](https://scryfall.com/search?q=!"Fynn,+the+Fangbearer")
- [x] Done -- [Gates of Istfell](https://scryfall.com/search?q=!"Gates+of+Istfell")
- [x] Done -- [Giant Ox](https://scryfall.com/search?q=!"Giant+Ox")
- [x] Done -- [Giant's Amulet](https://scryfall.com/search?q=!"Giants+Amulet")
- [x] Done -- [Giant's Grasp](https://scryfall.com/search?q=!"Giants+Grasp")
- [x] Done -- [Gilded Assault Cart](https://scryfall.com/search?q=!"Gilded+Assault+Cart")
- [x] Done -- [Glacial Floodplain](https://scryfall.com/search?q=!"Glacial+Floodplain")
- [x] Done -- [Gladewalker Ritualist](https://scryfall.com/search?q=!"Gladewalker+Ritualist")
- [x] Done -- [Glimpse the Cosmos](https://scryfall.com/search?q=!"Glimpse+the+Cosmos")
- [x] Done -- [Glittering Frost](https://scryfall.com/search?q=!"Glittering+Frost")
- [x] Done -- [Glorious Protector](https://scryfall.com/search?q=!"Glorious+Protector")
- [x] Done -- [Gnottvold Recluse](https://scryfall.com/search?q=!"Gnottvold+Recluse")
- [x] Done -- [Gnottvold Slumbermound](https://scryfall.com/search?q=!"Gnottvold+Slumbermound")
- [x] Done -- [Gods' Hall Guardian](https://scryfall.com/search?q=!"Gods+Hall+Guardian")
- [x] Done -- [Goldmaw Champion](https://scryfall.com/search?q=!"Goldmaw+Champion")
- [x] Done -- [Goldspan Dragon](https://scryfall.com/search?q=!"Goldspan+Dragon")
- [x] Done -- [Goldvein Pick](https://scryfall.com/search?q=!"Goldvein+Pick")
- [x] Done -- [Graven Lore](https://scryfall.com/search?q=!"Graven+Lore")
- [x] Done -- [Great Hall of Starnheim](https://scryfall.com/search?q=!"Great+Hall+of+Starnheim")
- [x] Done -- [Grim Draugr](https://scryfall.com/search?q=!"Grim+Draugr")
- [x] Done -- [Grizzled Outrider](https://scryfall.com/search?q=!"Grizzled+Outrider")
- [x] Done -- [Guardian Gladewalker](https://scryfall.com/search?q=!"Guardian+Gladewalker")
- [x] Done -- [Hagi Mob](https://scryfall.com/search?q=!"Hagi+Mob")
- [x] Done -- [Hailstorm Valkyrie](https://scryfall.com/search?q=!"Hailstorm+Valkyrie")
- [x] Done -- [Halvar, God of Battle](https://scryfall.com/search?q=!"Halvar,+God+of+Battle")
- [x] Done -- [Harald Unites the Elves](https://scryfall.com/search?q=!"Harald+Unites+the+Elves")
- [x] Done -- [Harald, King of Skemfar](https://scryfall.com/search?q=!"Harald,+King+of+Skemfar")
- [x] Done -- [Haunting Voyage](https://scryfall.com/search?q=!"Haunting+Voyage")
- [x] Done -- [Hengegate Pathway](https://scryfall.com/search?q=!"Hengegate+Pathway")
- [x] Done -- [Highland Forest](https://scryfall.com/search?q=!"Highland+Forest")
- [x] Done -- [Horizon Seeker](https://scryfall.com/search?q=!"Horizon+Seeker")
- [x] Done -- [Ice Tunnel](https://scryfall.com/search?q=!"Ice+Tunnel")
- [x] Done -- [Icebind Pillar](https://scryfall.com/search?q=!"Icebind+Pillar")
- [x] Done -- [Icebreaker Kraken](https://scryfall.com/search?q=!"Icebreaker+Kraken")
- [x] Done -- [Icehide Troll](https://scryfall.com/search?q=!"Icehide+Troll")
- [x] Done -- [Immersturm Predator](https://scryfall.com/search?q=!"Immersturm+Predator")
- [x] Done -- [Immersturm Raider](https://scryfall.com/search?q=!"Immersturm+Raider")
- [x] Done -- [Immersturm Skullcairn](https://scryfall.com/search?q=!"Immersturm+Skullcairn")
- [x] Done -- [In Search of Greatness](https://scryfall.com/search?q=!"In+Search+of+Greatness")
- [x] Done -- [Infernal Pet](https://scryfall.com/search?q=!"Infernal+Pet")
- [x] Done -- [Inga Rune-Eyes](https://scryfall.com/search?q=!"Inga+Rune-Eyes")
- [x] Done -- [Invasion of the Giants](https://scryfall.com/search?q=!"Invasion+of+the+Giants")
- [x] Done -- [Invoke the Divine](https://scryfall.com/search?q=!"Invoke+the+Divine")
- [x] Done -- [Iron Verdict](https://scryfall.com/search?q=!"Iron+Verdict")
- [x] Done -- [Jarl of the Forsaken](https://scryfall.com/search?q=!"Jarl+of+the+Forsaken")
- [x] Done -- [Jaspera Sentinel](https://scryfall.com/search?q=!"Jaspera+Sentinel")
- [x] Done -- [Jorn, God of Winter](https://scryfall.com/search?q=!"Jorn,+God+of+Winter")
- [x] Done -- [Kardur's Vicious Return](https://scryfall.com/search?q=!"Kardurs+Vicious+Return")
- [x] Done -- [Kardur, Doomscourge](https://scryfall.com/search?q=!"Kardur,+Doomscourge")
- [x] Done -- [Karfell Harbinger](https://scryfall.com/search?q=!"Karfell+Harbinger")
- [x] Done -- [Karfell Kennel-Master](https://scryfall.com/search?q=!"Karfell+Kennel-Master")
- [x] Done -- [Kaya the Inexorable](https://scryfall.com/search?q=!"Kaya+the+Inexorable")
- [x] Done -- [Kaya's Onslaught](https://scryfall.com/search?q=!"Kayas+Onslaught")
- [x] Done -- [King Harald's Revenge](https://scryfall.com/search?q=!"King+Haralds+Revenge")
- [x] Done -- [King Narfi's Betrayal](https://scryfall.com/search?q=!"King+Narfis+Betrayal")
- [x] Done -- [Koll, the Forgemaster](https://scryfall.com/search?q=!"Koll,+the+Forgemaster")
- [x] Done -- [Kolvori, God of Kinship](https://scryfall.com/search?q=!"Kolvori,+God+of+Kinship")
- [x] Done -- [Koma's Faithful](https://scryfall.com/search?q=!"Komas+Faithful")
- [x] Done -- [Koma, Cosmos Serpent](https://scryfall.com/search?q=!"Koma,+Cosmos+Serpent")
- [x] Done -- [Littjara Glade-Warden](https://scryfall.com/search?q=!"Littjara+Glade-Warden")
- [x] Done -- [Littjara Kinseekers](https://scryfall.com/search?q=!"Littjara+Kinseekers")
- [x] Done -- [Littjara Mirrorlake](https://scryfall.com/search?q=!"Littjara+Mirrorlake")
- [x] Done -- [Magda, Brazen Outlaw](https://scryfall.com/search?q=!"Magda,+Brazen+Outlaw")
- [x] Done -- [Maja, Bretagard Protector](https://scryfall.com/search?q=!"Maja,+Bretagard+Protector")
- [x] Done -- [Mammoth Growth](https://scryfall.com/search?q=!"Mammoth+Growth")
- [x] Done -- [Masked Vandal](https://scryfall.com/search?q=!"Masked+Vandal")
- [x] Done -- [Maskwood Nexus](https://scryfall.com/search?q=!"Maskwood+Nexus")
- [x] Done -- [Master Skald](https://scryfall.com/search?q=!"Master+Skald")
- [x] Done -- [Mists of Littjara](https://scryfall.com/search?q=!"Mists+of+Littjara")
- [x] Done -- [Mistwalker](https://scryfall.com/search?q=!"Mistwalker")
- [x] Done -- [Moritte of the Frost](https://scryfall.com/search?q=!"Moritte+of+the+Frost")
- [x] Done -- [Mystic Reflection](https://scryfall.com/search?q=!"Mystic+Reflection")
- [x] Done -- [Narfi, Betrayer King](https://scryfall.com/search?q=!"Narfi,+Betrayer+King")
- [x] Done -- [Niko Aris](https://scryfall.com/search?q=!"Niko+Aris")
- [x] Done -- [Niko Defies Destiny](https://scryfall.com/search?q=!"Niko+Defies+Destiny")
- [x] Done -- [Old-Growth Troll](https://scryfall.com/search?q=!"Old-Growth+Troll")
- [x] Done -- [Open the Omenpaths](https://scryfall.com/search?q=!"Open+the+Omenpaths")
- [x] Done -- [Orvar, the All-Form](https://scryfall.com/search?q=!"Orvar,+the+All-Form")
- [x] Done -- [Path to the World Tree](https://scryfall.com/search?q=!"Path+to+the+World+Tree")
- [x] Done -- [Pilfering Hawk](https://scryfall.com/search?q=!"Pilfering+Hawk")
- [x] Done -- [Poison the Cup](https://scryfall.com/search?q=!"Poison+the+Cup")
- [x] Done -- [Port of Karfell](https://scryfall.com/search?q=!"Port+of+Karfell")
- [x] Done -- [Priest of the Haunted Edge](https://scryfall.com/search?q=!"Priest+of+the+Haunted+Edge")
- [x] Done -- [Provoke the Trolls](https://scryfall.com/search?q=!"Provoke+the+Trolls")
- [x] Done -- [Pyre of Heroes](https://scryfall.com/search?q=!"Pyre+of+Heroes")
- [x] Done -- [Quakebringer](https://scryfall.com/search?q=!"Quakebringer")
- [x] Done -- [Raiders' Karve](https://scryfall.com/search?q=!"Raiders+Karve")
- [x] Done -- [Raise the Draugr](https://scryfall.com/search?q=!"Raise+the+Draugr")
- [x] Done -- [Rally the Ranks](https://scryfall.com/search?q=!"Rally+the+Ranks")
- [x] Done -- [Rampage of the Valkyries](https://scryfall.com/search?q=!"Rampage+of+the+Valkyries")
- [x] Done -- [Raven Wings](https://scryfall.com/search?q=!"Raven+Wings")
- [x] Done -- [Ravenform](https://scryfall.com/search?q=!"Ravenform")
- [x] Done -- [Ravenous Lindwurm](https://scryfall.com/search?q=!"Ravenous+Lindwurm")
- [x] Done -- [Realmwalker](https://scryfall.com/search?q=!"Realmwalker")
- [x] Done -- [Reckless Crew](https://scryfall.com/search?q=!"Reckless+Crew")
- [x] Done -- [Reflections of Littjara](https://scryfall.com/search?q=!"Reflections+of+Littjara")
- [x] Done -- [Reidane, God of the Worthy](https://scryfall.com/search?q=!"Reidane,+God+of+the+Worthy")
- [x] Done -- [Renegade Reaper](https://scryfall.com/search?q=!"Renegade+Reaper")
- [x] Done -- [Replicating Ring](https://scryfall.com/search?q=!"Replicating+Ring")
- [x] Done -- [Resplendent Marshal](https://scryfall.com/search?q=!"Resplendent+Marshal")
- [x] Done -- [Return Upon the Tide](https://scryfall.com/search?q=!"Return+Upon+the+Tide")
- [x] Done -- [Revitalize](https://scryfall.com/search?q=!"Revitalize")
- [x] Done -- [Righteous Valkyrie](https://scryfall.com/search?q=!"Righteous+Valkyrie")
- [x] Done -- [Rimewood Falls](https://scryfall.com/search?q=!"Rimewood+Falls")
- [x] Done -- [Rise of the Dread Marn](https://scryfall.com/search?q=!"Rise+of+the+Dread+Marn")
- [x] Done -- [Rootless Yew](https://scryfall.com/search?q=!"Rootless+Yew")
- [x] Done -- [Roots of Wisdom](https://scryfall.com/search?q=!"Roots+of+Wisdom")
- [x] Done -- [Run Amok](https://scryfall.com/search?q=!"Run+Amok")
- [x] Done -- [Run Ashore](https://scryfall.com/search?q=!"Run+Ashore")
- [x] Done -- [Rune of Flight](https://scryfall.com/search?q=!"Rune+of+Flight")
- [x] Done -- [Rune of Might](https://scryfall.com/search?q=!"Rune+of+Might")
- [x] Done -- [Rune of Mortality](https://scryfall.com/search?q=!"Rune+of+Mortality")
- [x] Done -- [Rune of Speed](https://scryfall.com/search?q=!"Rune+of+Speed")
- [x] Done -- [Rune of Sustenance](https://scryfall.com/search?q=!"Rune+of+Sustenance")
- [x] Done -- [Runed Crown](https://scryfall.com/search?q=!"Runed+Crown")
- [x] Done -- [Runeforge Champion](https://scryfall.com/search?q=!"Runeforge+Champion")
- [x] Done -- [Sarulf's Packmate](https://scryfall.com/search?q=!"Sarulfs+Packmate")
- [x] Done -- [Sarulf, Realm Eater](https://scryfall.com/search?q=!"Sarulf,+Realm+Eater")
- [x] Done -- [Saw It Coming](https://scryfall.com/search?q=!"Saw+It+Coming")
- [x] Done -- [Scorn Effigy](https://scryfall.com/search?q=!"Scorn+Effigy")
- [x] Done -- [Sculptor of Winter](https://scryfall.com/search?q=!"Sculptor+of+Winter")
- [x] Done -- [Search for Glory](https://scryfall.com/search?q=!"Search+for+Glory")
- [x] Done -- [Seize the Spoils](https://scryfall.com/search?q=!"Seize+the+Spoils")
- [x] Done -- [Shackles of Treachery](https://scryfall.com/search?q=!"Shackles+of+Treachery")
- [x] Done -- [Shepherd of the Cosmos](https://scryfall.com/search?q=!"Shepherd+of+the+Cosmos")
- [x] Done -- [Shimmerdrift Vale](https://scryfall.com/search?q=!"Shimmerdrift+Vale")
- [x] Done -- [Showdown of the Skalds](https://scryfall.com/search?q=!"Showdown+of+the+Skalds")
- [x] Done -- [Sigrid, God-Favored](https://scryfall.com/search?q=!"Sigrid,+God-Favored")
- [x] Done -- [Skemfar Avenger](https://scryfall.com/search?q=!"Skemfar+Avenger")
- [x] Done -- [Skemfar Elderhall](https://scryfall.com/search?q=!"Skemfar+Elderhall")
- [x] Done -- [Skemfar Shadowsage](https://scryfall.com/search?q=!"Skemfar+Shadowsage")
- [x] Done -- [Skull Raid](https://scryfall.com/search?q=!"Skull+Raid")
- [x] Done -- [Smashing Success](https://scryfall.com/search?q=!"Smashing+Success")
- [x] Done -- [Snakeskin Veil](https://scryfall.com/search?q=!"Snakeskin+Veil")
- [x] Done -- [Snow-Covered Forest](https://scryfall.com/search?q=!"Snow-Covered+Forest")
- [x] Done -- [Snow-Covered Island](https://scryfall.com/search?q=!"Snow-Covered+Island")
- [x] Done -- [Snow-Covered Mountain](https://scryfall.com/search?q=!"Snow-Covered+Mountain")
- [x] Done -- [Snow-Covered Plains](https://scryfall.com/search?q=!"Snow-Covered+Plains")
- [x] Done -- [Snow-Covered Swamp](https://scryfall.com/search?q=!"Snow-Covered+Swamp")
- [x] Done -- [Snowfield Sinkhole](https://scryfall.com/search?q=!"Snowfield+Sinkhole")
- [x] Done -- [Spectral Steel](https://scryfall.com/search?q=!"Spectral+Steel")
- [x] Done -- [Spirit of the Aldergard](https://scryfall.com/search?q=!"Spirit+of+the+Aldergard")
- [x] Done -- [Squash](https://scryfall.com/search?q=!"Squash")
- [x] Done -- [Stalwart Valkyrie](https://scryfall.com/search?q=!"Stalwart+Valkyrie")
- [x] Done -- [Starnheim Aspirant](https://scryfall.com/search?q=!"Starnheim+Aspirant")
- [x] Done -- [Starnheim Courser](https://scryfall.com/search?q=!"Starnheim+Courser")
- [x] Done -- [Starnheim Unleashed](https://scryfall.com/search?q=!"Starnheim+Unleashed")
- [x] Done -- [Story Seeker](https://scryfall.com/search?q=!"Story+Seeker")
- [x] Done -- [Strategic Planning](https://scryfall.com/search?q=!"Strategic+Planning")
- [x] Done -- [Struggle for Skemfar](https://scryfall.com/search?q=!"Struggle+for+Skemfar")
- [x] Done -- [Sulfurous Mire](https://scryfall.com/search?q=!"Sulfurous+Mire")
- [x] Done -- [Surtland Elementalist](https://scryfall.com/search?q=!"Surtland+Elementalist")
- [x] Done -- [Surtland Flinger](https://scryfall.com/search?q=!"Surtland+Flinger")
- [x] Done -- [Surtland Frostpyre](https://scryfall.com/search?q=!"Surtland+Frostpyre")
- [x] Done -- [Svella, Ice Shaper](https://scryfall.com/search?q=!"Svella,+Ice+Shaper")
- [x] Done -- [Tergrid's Shadow](https://scryfall.com/search?q=!"Tergrids+Shadow")
- [x] Done -- [Tergrid, God of Fright](https://scryfall.com/search?q=!"Tergrid,+God+of+Fright")
- [x] Done -- [The Bears of Littjara](https://scryfall.com/search?q=!"The+Bears+of+Littjara")
- [x] Done -- [The Bloodsky Massacre](https://scryfall.com/search?q=!"The+Bloodsky+Massacre")
- [x] Done -- [The Raven's Warning](https://scryfall.com/search?q=!"The+Ravens+Warning")
- [x] Done -- [The Three Seasons](https://scryfall.com/search?q=!"The+Three+Seasons")
- [x] Done -- [The Trickster-God's Heist](https://scryfall.com/search?q=!"The+Trickster-Gods+Heist")
- [x] Done -- [The World Tree](https://scryfall.com/search?q=!"The+World+Tree")
- [x] Done -- [Thornmantle Striker](https://scryfall.com/search?q=!"Thornmantle+Striker")
- [x] Done -- [Tibalt's Trickery](https://scryfall.com/search?q=!"Tibalts+Trickery")
- [x] Done -- [Toralf, God of Fury](https://scryfall.com/search?q=!"Toralf,+God+of+Fury")
- [x] Done -- [Tormentor's Helm](https://scryfall.com/search?q=!"Tormentors+Helm")
- [x] Done -- [Toski, Bearer of Secrets](https://scryfall.com/search?q=!"Toski,+Bearer+of+Secrets")
- [x] Done -- [Tundra Fumarole](https://scryfall.com/search?q=!"Tundra+Fumarole")
- [x] Done -- [Tuskeri Firewalker](https://scryfall.com/search?q=!"Tuskeri+Firewalker")
- [x] Done -- [Tyrite Sanctum](https://scryfall.com/search?q=!"Tyrite+Sanctum")
- [x] Done -- [Tyvar Kell](https://scryfall.com/search?q=!"Tyvar+Kell")
- [x] Done -- [Undersea Invader](https://scryfall.com/search?q=!"Undersea+Invader")
- [x] Done -- [Usher of the Fallen](https://scryfall.com/search?q=!"Usher+of+the+Fallen")
- [x] Done -- [Valki, God of Lies](https://scryfall.com/search?q=!"Valki,+God+of+Lies")
- [x] Done -- [Valkyrie Harbinger](https://scryfall.com/search?q=!"Valkyrie+Harbinger")
- [x] Done -- [Valkyrie's Sword](https://scryfall.com/search?q=!"Valkyries+Sword")
- [x] Done -- [Valor of the Worthy](https://scryfall.com/search?q=!"Valor+of+the+Worthy")
- [x] Done -- [Varragoth, Bloodsky Sire](https://scryfall.com/search?q=!"Varragoth,+Bloodsky+Sire")
- [x] Done -- [Vault Robber](https://scryfall.com/search?q=!"Vault+Robber")
- [x] Done -- [Vega, the Watcher](https://scryfall.com/search?q=!"Vega,+the+Watcher")
- [x] Done -- [Vengeful Reaper](https://scryfall.com/search?q=!"Vengeful+Reaper")
- [x] Done -- [Village Rites](https://scryfall.com/search?q=!"Village+Rites")
- [x] Done -- [Volatile Fjord](https://scryfall.com/search?q=!"Volatile+Fjord")
- [x] Done -- [Vorinclex, Monstrous Raider](https://scryfall.com/search?q=!"Vorinclex,+Monstrous+Raider")
- [x] Done -- [Waking the Trolls](https://scryfall.com/search?q=!"Waking+the+Trolls")
- [x] Done -- [Warchanter Skald](https://scryfall.com/search?q=!"Warchanter+Skald")
- [x] Done -- [Warhorn Blast](https://scryfall.com/search?q=!"Warhorn+Blast")
- [x] Done -- [Weathered Runestone](https://scryfall.com/search?q=!"Weathered+Runestone")
- [x] Done -- [Weigh Down](https://scryfall.com/search?q=!"Weigh+Down")
- [x] Done -- [Wings of the Cosmos](https://scryfall.com/search?q=!"Wings+of+the+Cosmos")
- [x] Done -- [Withercrown](https://scryfall.com/search?q=!"Withercrown")
- [x] Done -- [Woodland Chasm](https://scryfall.com/search?q=!"Woodland+Chasm")
- [x] Done -- [Youthful Valkyrie](https://scryfall.com/search?q=!"Youthful+Valkyrie")
</details>

**Comment by theelk801** [comment]
- [ ] In progress -- [Barkchannel Pathway](https://scryfall.com/search?q=!"Barkchannel+Pathway")
- [ ] In progress -- [Blightstep Pathway](https://scryfall.com/search?q=!"Blightstep+Pathway")
- [ ] In progress -- [Darkbore Pathway](https://scryfall.com/search?q=!"Darkbore+Pathway")
- [ ] In progress -- [Hengegate Pathway](https://scryfall.com/search?q=!"Hengegate+Pathway")
- [ ] In progress -- [Pyre of Heroes](https://scryfall.com/search?q=!"Pyre+of+Heroes")
- [ ] In progress -- [Showdown of the Skalds](https://scryfall.com/search?q=!"Showdown+of+the+Skalds")

[Scryfall gallery of everything currently unimplemented](https://scryfall.com/search?q=!"Barkchannel%20Pathway"or!"Blightstep%20Pathway"or!"Darkbore%20Pathway"or!"Hengegate%20Pathway"or!"Pyre%20of%20Heroes"or!"Showdown%20of%20the%20Skalds"&unique=cards)

**Comment by theelk801** [comment]
additional tasks: none yet

**Comment by weirddan455** [comment]
I'd like to help with this if you don't mind (or does your second post mean you're already working on all these cards?)  I don't think I have permissions to check stuff off on the list but I'm about to send a pull request for Absorb Identity.

**Comment by theelk801** [comment]
Totally, feel free to contribute. I’ve been busy so I haven’t really gotten into these yet.

**Comment by weirddan455** [comment]
I'm taking Fire Giant's Fury next (saw the list updating so wanted to make sure we're not taking the same cards.)

**Comment by theelk801** [comment]
Yeah, no problem. The help is much appreciated.

**Comment by weirddan455** [comment]
I'll take Thornmantle Striker unless you want it.

**Comment by theelk801** [comment]
It's all yours.

**Comment by weirddan455** [comment]
Lots of new cards spoiled today.  Is there a script we can run to fetch the card data so Utils/gen-card.pl will work for them?

I also just sent a PR implementing the Boast mechanic as a common ability class so that we can use that for the new cards.

**Comment by theelk801** [comment]
I have a personal script I use but I'm waiting on scryfall to update with all the new stuff. Usually I don't implement a new mechanic until I'm implementing a card that uses it.

**Comment by weirddan455** [comment]
I'm planning on implementing Varragoth, Bloodsky Sire which uses Boast.  I just don't know the formatting for Utils/mtg-cards-data.txt which is why I was asking about a script.  I guess it's probably best to wait on scryfall?

**Comment by theelk801** [comment]
If you want to implement it now you can, but scryfall should be caught up by the end of the day. The formatting is like this:

`Card Name|Set Name (Full)|Collector Number|Rarity (C/U/R/M)|Mana Cost|Type Line|Power|Toughness|Rules text with $ denoting line breaks|`

If it's a planeswalker then replace `|Power|Toughness|` with `|Loyalty|`. Also don't worry about having multiple versions, those will get added later.

**Comment by weirddan455** [comment]
Thanks.  I've got it implemented now.  I'll do the snow duals too.

**Comment by ciaccona007** [comment]
Just implemented Sigrid. I'm no expert on git so before I merge the PR I was wondering which procedure you guys prefer when merging them (squash & merge, rebase, merge commit).

**Comment by weirddan455** [comment]
I'm going to take Calamity Bearer.

**Comment by JayDi85** [comment]
@ciaccona007 depends on PR's commits: 
* If it contains a good commits for change history (one commit per feature or fix) then use merge;
* If it contains a multiple commits like "merge", "fix type", "fix another type", "forget something" then use squash with good description;
* If you want to put old commits on the top (if other devs must read it) then you can use rebase (example: old PR with important changes). Another use case: if you have a local commits and want to sync sources -- then use rebase update instead merge (IntelliJ IDEA have an easy to use settings for that);

P.S. If you want PR's review then don't merge it -- wait a new review from another dev. If you ensure about fix or card then you can add it to master without PR.

**Comment by weirddan455** [comment]
Taking Esika, God of the Tree next.

**Comment by weirddan455** [comment]
Taking Koll, the Forgemaster.

**Comment by weirddan455** [comment]
I've got Tyvar Kell mostly done.  I'm just missing the haste ability on the emblem which is worded strangely.  Just from a Magic rules perspective, I'm not sure how giving an Elf on the stack haste makes sense.

I'm thinking I may have to give the emblem a static ability so it can give haste once the Elf has actually entered the battlefield (after checking that it was cast) but if anyone has a better idea, let me know.

I'll take another look at it tomorrow and try to get it working.

**Comment by theelk801** [comment]
You can probably generate a continuous effect that applies to it on the stack and on the battlefield, you just need to check two possible zone change counter values.

**Comment by weirddan455** [comment]
I'll take Kolvori, God of Kinship.

**Comment by jeffwadsworth** [comment]
I can do Withercrown if it is available.  Just starting at the bottom of the list.

**Comment by weirddan455** [comment]
I'll take Battle of Frost and Fire.

**Comment by weirddan455** [comment]
Taking Draugr's Helm next.

**Comment by weirddan455** [comment]
I'm taking Firja's Retribution.

**Comment by weirddan455** [comment]
@jeffwadsworth You probably need to clear the database if cards don't show up.  I have a script that deletes the Mage.Client/db and Mage.Server/db folders that I run before I test a new card.

**Comment by JayDi85** [comment]
@weirddan455 server/client/tests runs a new cards checking on startup, so it will add new card to database automaticly. DB cleaning can be useful only for card's updated text manual checking.

More tips:
* If you want to check actual card texts then run unit test `VerifyCardDataTest->test_showCardInfo` -- it's very fast and don't use db;
* If you want to speed up a unit tests start up time then disable new cards checking by  `FAST_SCAN_WITHOUT_DATABASE_CREATE` param (search it in code);

**Comment by theelk801** [comment]
Yeah it's good to write unit tests whenever you can, even though I don't do it nearly enough.

**Comment by theelk801** [comment]
@jeffwadsworth you should be able to use the same code as [Leonin Shikari](https://scryfall.com/card/c17/66/leonin-shikari) for Cosmos Charger.

**Comment by jeffwadsworth** [comment]
> @jeffwadsworth you should be able to use the same code as [Leonin Shikari](https://scryfall.com/card/c17/66/leonin-shikari) for Cosmos Charger.

Looks promising.  Thanks.

https://github.com/magefree/mage/blob/master/Mage/src/main/java/mage/abilities/effects/common/continuous/ActivateAbilitiesAnyTimeYouCouldCastInstantEffect.java

**Comment by weirddan455** [comment]
Just saw you pushed the snow land collation.  Is there something we need to do to enable booster draft for the set seeing as it's mostly complete now?

**Comment by theelk801** [comment]
I don't think so but I don't actually know for sure

**Comment by weirddan455** [comment]
@theelk801 OK, it seems to be working now.  I thought I had tried it a week or 2 ago and KHM packs weren't an option but maybe I just missed it.

**Comment by theelk801** [comment]
I'll take Cosima

**Comment by jeffwadsworth** [comment]
A proposed change to the way ForetellAbility works.  It was required to get Dream Devourer to work correctly.  Does anyone see any issues with it?  It works so far in my testing, though the text on the card is not correct yet...no doubt that can be fixed.
[ForetellAbility.txt](https://github.com/magefree/mage/files/5891708/ForetellAbility.txt)

**Comment by theelk801** [comment]
Thanks to everyone who helped get this done.

**Comment by JayDi85** [comment]
Visibility of MDF card's counters was fixed in c381ea77168c382c365de197d670aae7051b9ba4 (Cosima, God of the Voyage), so it can be closed.
